### PR TITLE
Dao 7

### PR DESCRIPTION
**Pull request by entertainment-veks**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -24,6 +25,12 @@
             <artifactId>mysql-connector-java</artifactId>
             <version>8.0.22</version>
         </dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
+            <version>3.0.1</version>
```
Comment (Fedich09) [inline pom.xml:31]: we shouldn't use alpha version 
```suggestion
            <version>3.0.1</version>
```

File: src/main/java/taxiservice/service/CarService.java [hunk 2]
```diff
@@ -4,15 +4,20 @@
...
     Car get(Long id);
 
+    @Override
     List<Car> getAll();
 
+    @Override
     Car update(Car car);
 
+    @Override
     boolean delete(Long id);
```
Comment (Fedich09) [inline src/main/java/taxiservice/service/CarService.java:21]:
```suggestion

```

File: src/main/java/taxiservice/service/DriverService.java [hunk 3]
```diff
@@ -3,14 +3,19 @@
...
     Driver get(Long id);
 
+    @Override
     List<Driver> getAll();
 
+    @Override
     Driver update(Driver driver);
 
+    @Override
     boolean delete(Long id);
```
Comment (Fedich09) [inline src/main/java/taxiservice/service/DriverService.java:20]:
```suggestion

```
we implement this methods in CarImpl , DriverImpl and we created GenericServise with all CRUD methods, you shouldn't override this

File: src/main/java/taxiservice/service/ManufacturerService.java [hunk 4]
```diff
@@ -3,14 +3,19 @@
...
     Manufacturer get(Long id);
 
+    @Override
     List<Manufacturer> getAll();
 
+    @Override
     Manufacturer update(Manufacturer manufacturer);
 
+    @Override
     boolean delete(Long id);
```
Comment (Fedich09) [inline src/main/java/taxiservice/service/ManufacturerService.java:20]:
```suggestion

```

File: src/main/webapp/web.xml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+        <servlet-name>index</servlet-name>
+        <servlet-class>taxiservice.controllers.IndexController</servlet-class>
+    </servlet>
+    
+    <servlet-mapping>
+        <servlet-name>index</servlet-name>
+        <url-pattern>/index</url-pattern>
+    </servlet-mapping>
+
+</web-app>
```
Comment (Fedich09) [inline src/main/webapp/web.xml:17]: add empty line please

File: src/main/webapp/WEB-INF/views/index.jsp [hunk 6]
```diff
@@ -0,0 +1,16 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: egorka
+  Date: 20.01.2021
+  Time: 11:00
+  To change this template use File | Settings | File Templates.
+--%>
```
Comment (Fedich09) [inline src/main/webapp/WEB-INF/views/index.jsp:7]: delete this comment please

**Review by Fedich09 — CHANGES_REQUESTED** [review]

**Review by Fedich09 — APPROVED** [review]

**Review by boroda4436 — CHANGES_REQUESTED** [review]
There are some classes from previous HW presented. Please remove not required files from this PR

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]
let's close this PR and make a new one that does not contain redundant commits

**Review by boroda4436 — COMMENTED** [review]
Бачу ти видалив всі сервіси.
А що буде коли ти замерджиш цей код?

**Pull request closed without merging**